### ISSUE TITLE
Show version number

### ISSUE THREAD

**Issue by hectoryo** [Xeroday/Spotify-Ad-Blocker#199]
Currently I have the latest version of EZBlocker, but when I open it, I don't see anywhere the version number of the program I'm running.
Could it be possible to add the version number in the window or maybe in an "About" window?

**Comment by DerRockWolf** [comment]
Good idea. In the meantime you can click on "Report Problem". It will show current EZBlocker version and current Spotify version and it will also copy them to the clipboard.
